Report.py

```diff
@@ -1164,7 +1167,12 @@ def writeHtmlFile(self, baseNameBeforeExtension, tree, reportSummary):
             ((self.filing.transform, self.filing.fileNameBase),) + (
             ((self.filing.transformDissem, self.filing.dissemFileNameBase),) if self.filing.transformDissem else ())):
             _startedAt = time.time()
-            result = _transform(tree, asPage=XSLT.strparam('true'))
+            cell_count = sum(1 for x in tree.iter('Cell'))
+            if cell_count > 50000: 
+                self.controller.logWarn(f"There are {cell_count} cells; skipping transformation.")
+                result = fromstring("<HTML><HEAD><TITLE>NOPE</TITLE></HEAD><BODY>NOT TODAY FRIEND</BODY></HTML>")

```

Comment:
Is `<BODY>NOT TODAY FRIEND</BODY>` meant to go to production?

Reply:
Walter sourced that, changes?